roles/kubernetes-apps/local_volume_provisioner/templates/configmap.yml.j2

```diff
@@ -7,7 +7,8 @@ metadata:
   name: local-volume-config
   namespace: {{ system_namespace }}
 data:
-  storageClassMap: |
```

Comment:
All exemples that seems up-to-date are using 
```
storageClassMap: | 
   {{storage_class_name}}:
         hostDir:
        mountDir:
```
- https://github.com/kubernetes-incubator/external-storage/blob/00358906a211af2bcbe2479b2cf47258905e4738/local-volume/provisioner/pkg/common/common.go#L137-L139


- https://github.com/kubernetes-incubator/external-storage/blob/6180663533cfe6dc26c387e2131df92e05c2caf9/local-volume/provisioner/deployment/kubernetes/gce/provisioner_generated_gce_ssd_count.yaml#L10-L13

- https://github.com/kubernetes-incubator/external-storage/blob/6180663533cfe6dc26c387e2131df92e05c2caf9/local-volume/provisioner/deployment/kubernetes/gce/provisioner_generated_gce_ssd_volumes.yaml#L10-L16

- https://github.com/kubernetes-incubator/external-storage/blob/701f2e55155a8d13773d35df8dc08bdb59012764/local-volume/helm/provisioner/templates/provisioner.yaml#L51-L56

Reply:
@ant31 Please make sure you're looking at the local-volume-provisioner-v1.0.1 tag since that's what kubespray is using.

Reply:
thanks,  why not updating the tag?

Reply:
probably a mistake. 
So if the templates are correct using latest release, I'd say let's update the image tag rather than fixing the templates.

Reply:
Not sure how much more would need to change. That code base is quite different. I don’t have a lot more time I’m able to spend on this issue either unfortunately. We’ve fixed it in our private copy of Kubespray with the proposed fix and are continuing on. Sorry. :(

Reply:
@mirwan I propose to merge this and to open an issue to upgrade to v2.0.0 on a separate PR

Reply:
@ant31 Thanks